integration-tests/__tests__/__snapshots__/failures.test.js.snap

```diff
@@ -246,12 +242,11 @@ exports[`not throwing Error objects 5`] = `
       38 | 
       39 | test('returned promise rejection', () => {
 
-      at packages/jest-jasmine2/build/jasmine/Env.js:537:34
       at __tests__/during_tests.test.js:36:3
 
   ● returned promise rejection
 
-    Failed: Object {
+    thrown: Object {
```

Comment:
this one will probably make CI fail...

Reply:
stderr.replace(/thrown|Failed/, 'REPLACED<Error>');
😛